server/src/main/java/org/spine3/server/aggregate/AggregateStorage.java

```diff
@@ -62,19 +59,19 @@ protected AggregateStorage(boolean multitenant) {
     }
 
     @Override
-    public Optional<AggregateEvents> read(I aggregateId) {
+    public Optional<AggregateStateRecord> read(I aggregateId) {
         checkNotClosed();
         checkNotNull(aggregateId);
 
         final Deque<Event> history = newLinkedList();
         Snapshot snapshot = null;
 
-        final Iterator<AggregateStorageRecord> historyBackward = historyBackward(aggregateId);
+        final Iterator<AggregateEventRecord> historyBackward = historyBackward(aggregateId);
 
         while (historyBackward.hasNext()
                 && snapshot == null) {
 
-            final AggregateStorageRecord record = historyBackward.next();
+            final AggregateEventRecord record = historyBackward.next();
```

Comment:
May it be just `AggregateRecord`? We store not only events, but also snapshots in this form.